### PR TITLE
New debian release

### PR DESCRIPTION
**Pull request by unera**
Hi!

I've just uploaded 2.6 to Debian and did some changes (actual debian infrastructure)

Please check it

### CODE REVIEW

File: debian/rules [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/make -f
 
-
+INIT_VARIANT	?= systemd
 VERSION  := $(shell dpkg-parsechangelog|grep ^Version|awk '{print $$2}')
 UVERSION := $(shell echo $(VERSION)|sed 's/-[[:digit:]]\+$$//')
 
-ifneq ($(wildcard /usr/bin/dh_systemd_start),)
+ifeq ($(INIT_VARIANT),systemd)
```
Comment (Totktonada) [inline debian/rules:7]: Maybe now is good time to just enable it unconditionally. All Debian and Ubuntu distros, which offers something else by default, already gone.

**Review by Totktonada — COMMENTED** [review]
Thank you for the PR! Please, consider several comments below.

I would also appreciate if you'll prefix the commit header with `build` prefix. It eases navigation over the git history.

BTW, can you give some suggestions how we can keep tarantool more or less up-to-date in exisitng Debian releases?

**Comment by kyukhin** [comment]
No reaction from submitter for more than 3 months.

**Comment by Totktonada** [comment]
Anyway I'll extract the proposed enhancements in a non-breaking way when time will permit.

**Comment by Totktonada** [comment]
NB: The debhelper version was bumped in PR #5638. We still need to look / verify / describe the remaining changes that are proposed here.

**Pull request closed without merging**